### DISCUSSION TITLE
みなさまからいただいた意見をIssue化するフローについて

### DISCUSSION THREAD

**Discussion by kskwtnk** [increments/qiita-discussions#19] [アナウンス]
たくさんのDiscussionをオープンしていただきありがとうございます！

いただいた意見をIssueとして可視化し、進捗をお伝えしたいと思っているのですが

- いただいた意見が、Qiitaを開発しているプライベートリポジトリ内でIssue化されている
- 意見が割れそうなDiscussionの場合、どのタイミングでIssue化するのがベストなのか運営として確立できていない

上記のような状態であるため、しばらくの間は「検討します」と言っているもののこのリポジトリ内でIssue化されない……といった状況になってしまうかもしれません。

途中でやり方が変わったりする可能性はあるのですが、できるだけ早く可視化できるように整えていきますので、まずはこういった意思があることをアナウンスさせていただきます！

**Comment by uhooi** [comment]
すでに Internal Repository で Issue 化されているのは、ユーザーにわかると嬉しいです。
いろいろ方法はあると思いますが、COCOA のように Internal ID を Issue に振ると運営側も管理しやすいと思います。
ref: https://github.com/cocoa-mhlw/cocoa/issues/290
